src/features/create-quiz/create-quiz-second-card/components/answer-item.tsx

```diff
@@ -53,16 +57,42 @@ export const AnswerItem: React.FC<AnswerItemProps> = ({
       </div>
       {getError(`${name}`)}
 
-      <div className="field-radiobutton mt-2">
-        <RadioButton
-          inputId={inputId}
-          name={name}
-          value={value}
-          onChange={() => handleRadioChange(index)}
-          checked={checked}
-        />
-        <label htmlFor={inputId}>This is the correct answer</label>
-      </div>
+      {questionType?.name?.toLowerCase() === "single answer" && (
```

Comment:
I suggest using a global constant for the question types instead of using strings.